tests/integration/test_signals.py

```diff
@@ -168,27 +178,27 @@ def test_new_pipeline_gets_added_to_open_release(self, my_scm_pipeline_run, appl
         overrides = {"PIPELINE_CHANGE_NOTIFICATION_SESSION": session}
         with override_settings(**overrides), username_on_model(SCMPipelineRun, "signal_tester"):
             pipeline_run = SCMPipelineRun.objects.create(application=application)
+            pipeline_run.status = constants.PIPELINE_STATUS_IN_PROGRESS
+            pipeline_run.save()
 
         assert release.scm_pipeline_runs.count() == before + 1
         assert release.scm_pipeline_runs.filter(pk__exact=my_scm_pipeline_run.pk).count() == 1
         assert release.scm_pipeline_runs.filter(pk__exact=pipeline_run.pk).count() == 1
 
     def test_not_added_to_release_when_skipped(self, my_scm_pipeline_run, application):
         before = SCMRelease.objects.count()
-        release = SCMRelease.objects.first()
-        assert len(release.scm_pipeline_runs.all()) == 1
 
         session = mock.MagicMock()
         overrides = {"PIPELINE_CHANGE_NOTIFICATION_SESSION": session}
         with override_settings(**overrides), username_on_model(SCMPipelineRun, "signal_tester"):
             pipeline_run = SCMPipelineRun.objects.create(application=application, status="skipped")
 
+        pipeline_run = SCMPipelineRun.objects.get(pk=pipeline_run.public_identifier)
```

Comment:
Is this line necessary?

Reply:
not really, it was a left over from debug. Removed